### PR TITLE
docs: update ansible-docker-podman.md

### PR DESCRIPTION
**Pull request by jenworthington**
editorial review, first pass

### CODE REVIEW

File: docs/gettingstarted/ansible-docker-podman.md [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
 
-All you need to do before running sc4s with Ansible is providing `env_file`. In the env file provide at least proper Splunk endpoint and HEC token.
-Create a file in `ansible/resources` catalog or edit [example file](/ansible/resources/env_file).
+1. Before running SC4S with Ansible, provide `env_file` with your Splunk endpoint and HEC token. To do this, create the file in `ansible/resources` catalog or edit the [example file](/ansible/resources/env_file):
 
 ``` dotenv
 --8<---- "ansible/resources/env_file"
 ```
-Next provide a host on which you want to run Docker Swarm cluster and host application in inventory file:
+2. Provide a list of hosts on which you want to run your Docker Swarm cluster and the host application in the inventory file:
```
Comment (mstopa-splunk) [inline docs/gettingstarted/ansible-docker-podman.md:10]:
```suggestion
2. Provide a list of hosts on which you want to run your cluster and the host application in the inventory file:
```

File: docs/gettingstarted/ansible-docker-podman.md [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
+1. If you have Ansible installed on your host, run the Ansible playbook to deploy SC4S. Otherwise, use the Docker Ansible image provided in the package:
 ```bash
 # From repository root
 docker-compose -f ansible/docker-compose.yml build
 docker-compose -f ansible/docker-compose.yml up -d
 docker exec -it ansible_sc4s /bin/bash
 ```
-Once you are in containers remote shell you can run Docker Swam ansible playbook.
-If you are authenticating via username/password:
+2. If you used the Docker Ansible image in step 1, then from your container remote shell, authenticate to and run the Docker Swam Ansible playbook.
```
Comment (mstopa-splunk) [inline docs/gettingstarted/ansible-docker-podman.md:22]: Name collision, step 1 was used in the header of the previous section

Comment (mstopa-splunk) [inline docs/gettingstarted/ansible-docker-podman.md:22]: `and run the playbook`

File: docs/gettingstarted/ansible-docker-podman.md [hunk 3]
```diff
@@ -52,19 +48,19 @@ This should yield an event similar to the following:
 ```ini
 syslog-ng starting up; version='3.28.1'
 ```
-You can verify if all services in swarm cluster are working by checking ```sc4s_container``` field in splunk- each service should be recognized by different container id. All other fields should be the same.
+You can verify if all services in the Swarm cluster work by checking the ```sc4s_container``` in Splunk. Each service should have a different container ID. All other fields should be the same.
```
Comment (mstopa-splunk) [inline docs/gettingstarted/ansible-docker-podman.md:51]:
```suggestion
You can verify if all SC4S instances work by checking the ```sc4s_container``` in Splunk. Each instance should have a different container ID. All other fields should be the same.
```

**Comment by mstopa-splunk** [comment]
@jenworthington ready for the final pass

File: docs/gettingstarted/ansible-docker-podman.md (hunk 1 above)
Comment (jenworthington) [reply docs/gettingstarted/ansible-docker-podman.md:10]: updated

File: docs/gettingstarted/ansible-docker-podman.md (hunk 2 above)
Comment (jenworthington) [reply docs/gettingstarted/ansible-docker-podman.md:22]: good point, that is totally confusing! I changed it to "previous step" but I'm making notes and maybe we can rethink the numbering and structure oin the next round of reviews.

Comment (jenworthington) [reply docs/gettingstarted/ansible-docker-podman.md:22]: updated

**Pull request merged**